### ISSUE TITLE
Axis labels not updated on changing data in production mode

### ISSUE THREAD

**Issue by mi01** [plouc/nivo#1307]
Hello,
thank you for developing Nivo, i really like this library. But i found a bug in the current version of the `ResponsiveBar` component, which occurs only in production mode (`npm run build`). When downgrading to 0.63.1 the problem is gone.

I created a codesandbox [here](https://codesandbox.io/s/pedantic-goldberg-np7ut), but as is said, in development mode the problem does not occur. So i deployed the example also to Netlify [here](https://csb-np7ut.netlify.app/). Its quite obvious whats the problem: The labels of the axis at the bottom are not moving, when changing the dataset.

Not sure if this is related, but in the browser console i get the following error in production mode:

```
TypeError: r.willAdvance is not a function
    advance FrameLoop.ts:173
    cc React
    advance FrameLoop.ts:169
    n FrameLoop.ts:107
FrameLoop.ts:110:16
```

**Expected behavior**
Expecting the axis labels to adjust to the updated data list.

**Comment by wyze** [comment]
Duplicate of #1290. It is a bug within react-spring that we are waiting on a new version to get released with a fix. https://github.com/pmndrs/react-spring/issues/1078

There are some workarounds in the linked issue, we also have a script we use for our website to work around it. https://github.com/plouc/nivo/blob/master/scripts/patch-react-spring.js

**Comment by plouc** [comment]
Closing in favor of #1290
